packages/react/src/components/DataTable/DataTable.js

```diff
@@ -105,6 +105,11 @@ export default class DataTable extends React.Component {
      */
     translateWithId: PropTypes.func,
 
+    /**
+     * `normal` Change the row height of table
+     */
+    size: PropTypes.oneOf(['compact', 'short', 'normal', 'tall']),
```

Comment:
Would we rather this prop just live on the toolbar instead of being something passed in from DataTable with `getToolbarProps`?

Also, seems like the comment is related to row size changes potentially?

Reply:
As the matter of fact, [`<DataTable>` has `size` prop already](https://github.com/carbon-design-system/carbon/blob/v10.10.0/packages/react/src/components/DataTable/DataTable.js#L349) and thus I thought it's sensible to propagate that to the tool bar. Let us know if you still like letting user set the toolbar size in application code, or something else.

Reply:
@asudoh got it, makes sense!